semcore/accordion/src/index.d.ts

```diff
@@ -52,6 +58,10 @@ export type AccordionItemProps = {
   /** Animation duration
    * @default 350 */
   duration?: number;
+  /**
+   * Changes the visual appearance of the item
+   */
+  compact?: boolean;
```

Comment:
not sure, that this is public property... I think all items in Accordion should be the same